panda/src/wgldisplay/wglGraphicsStateGuardian.cxx

```diff
@@ -611,8 +611,10 @@ make_context(HDC hdc) {
       attrib_list[n++] = WGL_CONTEXT_DEBUG_BIT_ARB;
     }
 #ifndef SUPPORT_FIXED_FUNCTION
-    attrib_list[n++] = WGL_CONTEXT_PROFILE_MASK_ARB;
-    attrib_list[n++] = WGL_CONTEXT_CORE_PROFILE_BIT_ARB;
+    if(!has_fixed_function_pipeline()) {
+      attrib_list[n++] = WGL_CONTEXT_PROFILE_MASK_ARB;
+      attrib_list[n++] = WGL_CONTEXT_CORE_PROFILE_BIT_ARB;
+    }
```

Comment:
Sure, my fault! Regarding checking gl_version, I guess we better not check nothing, not at least yet...
As far as I know, almost all the GL implementations are automatically offering the latest compatibility GL context (i.e. 3.1 in case of only core-profile systems). And we can manually enforce a newer one just simply setting gl-version in Config.prc
Maybe in the future we can think about implementing another logic to generate the GL contexts. Maybe a core profile flag at Config.prc??